plugins/keep-markup/tests/test.js

```diff
@@ -0,0 +1,92 @@
+const expect = require('chai').expect;
+const jsdom = require('jsdom')
+const { JSDOM } = jsdom
```

Comment:
This destructuring is what's breaking the tests, as Node 4 didn't support this syntax. Maybe we don't need to test on Node 4?

Reply:
Node 4 is EOL since 30/04/2018: https://github.com/nodejs/Release
I think we should set a minimum version in the `package.json`:

```js
{
  "engines": {
    "node": ">=8.11",
    "npm": ">=5.0.0"
  }
}
```

I'm using the following configuration in `.travis.yml` to test against LTS (and the latest stable release of Node):

```
node_js:
- 'lts/*'
- 'node'
```

Reply:
It's that config even better than #1559?

Reply:
I guess since you don't have to update the version of Node when a new LTS comes out.
I don't think we should test against EOL versions of Node.js.

In my opinion `lts/*` + `node` is a good default. But if you want to build against Node 6 and Node 8 I'm fine :smile:

Reply:
Seems like I misunderstood the meaning of [`lts/*`](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions). I thought it includes all LTS releases of NodeJS given the `*` suffix. My bad.